### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by riyasmjgeo** [gmtsar/gmtsar#370] [Q&A] [answered]
Hello All,

I am unable to install GMTSAR even after multiple tries. 

I have attached three files 
1. installs.txt: to show the required packages are successfully installed.
2. Till_make.txt: Command lines till making!
3. make_issue: The error on making.
[installs.txt](https://github.com/gmtsar/gmtsar/files/9110768/installs.txt)
[make_issue.txt](https://github.com/gmtsar/gmtsar/files/9110769/make_issue.txt)
[Till_make.txt](https://github.com/gmtsar/gmtsar/files/9110770/Till_make.txt)

The error states that there are multiple definitions of some variables. Pls, share your thoughts on this.
I tried to configure without the orbits, tried the current GitHub version, and tried installing GMTSAR after reinstalling all the required packages. None of the tries worked, and I am stuck on the same issue.
OS: Ubuntu 22.04
GMT: 6.3.0 [64-bit]
GMTSAR Version: Cloned Git branch 6.2

Thank you.
RMJ

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
This might be the case? Adding -z muldefs to the CFLAG and LDFLAG should solve the problem.
https://github.com/gmtsar/gmtsar/issues/165

**Reply by riyasmjgeo** [reply]
Right on spot. Thank you so much.
